packages/Pagination/src/Pagination.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import ArrowItem from "./components/ArrowItem/ArrowItem";
+import CurrentPageItem from "./components/CurrentPageItem/CurrentPageItem";
+import ElipsisItem from "./components/EllipsisItem/ElipsisItem";
+import PageItem from "./components/PageItem/PageItem";
+import Divider from "./components/Divider";
+import * as sc from "./Pagination.styles";
+
+const propTypes = {
+  /** The number of current active page */
+  currentPage: PropTypes.number.isRequired,
+  /** Length of the pagination (the number of other pages will be displayed around active page). */
+  length: ShirtSizes,
```

Comment:
🤔 don't know if this prop is clear about what actually does... what does length do? and why has `shirtsize` attached to it? this is why I think micro abstraction like `shirtsize` doesn't escalate properly, doesn't have any meaning by itself, needs a prop to create one, and in this case, the prop doesn't help too much in providing a meaningful meaning. 

If we were using something like [types.small, types.big] then this would make more sense but `ShirtSize` is a term that is too abstract to be understood. 

cc @tristanjasper

Reply:
For this case can we use a more meaningful prop name? 🙏